### PR TITLE
Feature/bash examples.

### PR DESCRIPTION
**Pull request by ejeliot**

### CODE REVIEW

File: source/includes/_batch.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ if ($result['success']) {
 }
 ```
 
+```shell
+curl -X POST \
+    -F 'api-key=[INSERT_API_KEY]' \
```
Comment (dmytro-vovk) [inline source/includes/_batch.md]: Aren't the corner parentheses more appropriate? `api-key=<INSERT_API_KEY>`

Comment (ejeliot) [inline source/includes/_batch.md]: Yes, fair point. Changing.

File: source/includes/_lsrc.md [hunk 2]
```diff
@@ -149,23 +149,23 @@ campaign-id | <span class="label label-required">Required</span>
                 "schedule": "Weekly",
                 "day_of_week": "Thursday",
                 "day_of_month": "0",
-                "location_id": "0",
+                "location_id": "0"
```
Comment (dmytro-vovk) [inline source/includes/_lsrc.md:152]: String or integer?

Comment (ejeliot) [inline source/includes/_lsrc.md:152]: Rightly or wrongly is returned as string.

**Pull request merged**